### PR TITLE
Lesson3 modified

### PR DESCRIPTION
**Pull request by yyht6300a**
- Separated some steps into small parts for giving clear guidance.
- Provided extra examples to show how functions work.

### CODE REVIEW

**Comment by Ansper11** [comment]
It looks nice, thanks

File: webserver/userFolder/lesson3.json [hunk 1]
```diff
@@ -27,43 +27,79 @@
...
-        "code": "Map = [\r\n    ('A','B',1),('A','C',2),\r\n    ('B','C',6),('B','D',11),\r\n    ('C','D',9),('C','E',13),\r\n    ('D','E',7),('D','F',3),\r\n    ('E','F',4)\r\n    ]\r\nnodes = ('ABCDEF')\r\nMap.sort(key=lambda x:x[2])\r\n'''\r\ntree=dict()\r\nfor loop for all elements in nodes:\r\n    let each node connect with itself (we will deal with this later.)\r\n'''\r\ntrees=dict()\r\nfor i in nodes:\r\n    trees[i]=i\r\nprint(Map)\r\nprint(trees)",
+        "instructions": "<h1>Step 3:</h1><br/> <h3>In this part, we need to sort edges in increasing order by using sort(). <br/>In python, function sort() looks like<br/> sort(key=None,reverse=False)<br/>key is the compaire function. In this part, we will use key=lambda s: s[t], where s means each iteration and t means the column. </h3> ",
```
Comment (Nick-McGee) [inline webserver/userFolder/lesson3.json:30]: Thank you for cleaning this up a bit.

**Comment by yyht6300a** [comment]
Thanks for the comments.

**Comment by yyht6300a** [comment]
I'll try to create a new Minecraft world next week

**Pull request merged**